TTHAnalysis/python/plotter/mcPlots.py

```diff
@@ -1187,6 +1187,8 @@ def addPlotMakerOptions(parser, addAlsoMCAnalysis=True):
     outname  = options.out if options.out else (args[2].replace(".txt","")+".root")
     if (not options.out) and options.printDir:
         outname = options.printDir + "/"+os.path.basename(args[2].replace(".txt","")+".root")
+    if '/pool/ciencias' in outname:
+        outname = outname.replace('/pool/ciencias/','/pool/cienciasrw/')
```

Comment:
Short answer: this is unfortunately still necessary.
Long answer: for now we are stuck with this system in which we have to read using /pool/ciencias/ and write using /pool/cienciasrw/ (the latter being the same partition mounted R/W but on a far slower connection than the one mounted in /pool/ciencias). We will test with a lustre system probably in January. I know it is crazy, but this is what we have now.